source/Octopus.Manager.Tentacle.Tests/ApplicationStartupSanityCheckFixture.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using NUnit.Framework;
+
+namespace Octopus.Manager.Tentacle.Tests
+{
+    [TestFixture]
+    public class ApplicationStartupSanityCheckFixture
+    {
+        [Test]
+        public void ApplicationCanStartWithoutCrashing()
+        { 
+            Exception threadException = null;
+            var thread = new Thread(() =>
```

Comment:
I'm still a bit wary about this.

It definitely tests what we want to test, and works well when I run it locally, but there is a lot of thread-wrangling just to open a UI window. Most of the functionality being tested here would also be tested by resolving the `TentacleManagerModel`.

Let's keep the test as-is, but also keep an eye on it in the short term. We are still going to add some new tests that will resolve the view models, so as long as this test is reliable on the build server, it will be a net positive to have it.